src/style.css

```diff
@@ -0,0 +1,34 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
```

Comment:
Removing these lines in your demo in dev tools doesn't change anything for me, I think it's redundant.